File: Conditionals/time_of_day.py
# Ali Walton, Time of Day Python

import time
current = time.time()
local_time = time.localtime(current)
hour = local_time.tm_hour


if  hour >=12 and hour < 17 :
    print("Good afternoon!")
elif hour >= 17 :
     print("Good Evening!")
else:
    print("Good morning!")

File: Conditionals/time_of_day.c
//Ali Walton, Time of Day in C
#include <stdio.h>
#include <time.h>

int main(void){
    time_t rawtime;
    struct tm * timeinfo;
    time(&rawtime);
    timeinfo = localtime(&rawtime);
    time_t now = time(NULL);
    struct tm *tm_struct = localtime(&now);
    int hour = tm_struct->tm_hour;


    if  (hour >= 12 && hour < 17){
    printf("Good afternoon!\n");
    }else if (hour >=17){
    printf("Good evening!\n");
    }else{
    printf("Good morning!\n");
    }
    return 0;
}